src/cljs/rems/administration/form.cljs

```diff
@@ -75,8 +49,8 @@
               [inline-info-field (text :t.administration/start) (localize-time (:start form))]
               [inline-info-field (text :t.administration/end) (localize-time (:end form))]
               [inline-info-field (text :t.administration/active) [readonly-checkbox (not (:expired form))]]]}]
-   [form-fields (:fields form) language]
-   [:div.col.commands [back-button]]])
+   [:div.col.commands [back-button]]
```

Comment:
back is between the info and the preview. could also be at the top I guess. I don't want to have it below the preview, and I don't think a collapsible preview would be any better